example/spec/helpers/testUtils.js

```diff
@@ -100,7 +101,8 @@ function updateAndUploadTestFileToBucket(file, bucket, prefix = 'cumulus-test-da
   return s3().putObject({
     Bucket: bucket,
     Key: `${prefix}/${key}`,
-    Body: data
+    Body: data,
+    ContentType: mime.lookup(key) || 'binary/octet'
```

Comment:
Infer the mime-type from `key`. If that returns false, then use a default `mime-type` of `binary/octet`.

Writing this I realize that a better implementation would not hard-code a default value. I'm going to change this.

Reply:
Note that the default `ContentType` from `s3().putObject` is `binary/octet`.